elearning/overview/intro.rst

```diff
@@ -0,0 +1,55 @@
+=========
+Elearning
+=========
+
+Online learning has been on the rise in recent years, and it’s not really hard to see why.
+eLearning courses are more convenient than traditional face to face courses. Students can fit them around their existing
+responsibilities and commitments, allowing them to engage with content and extra learning materials whenever is most
+convenient to them.
+And not only that: students can save time commuting as they can simply log in to their virtual classes from the comfort
+of their home or office!
+
+Odoo Elearning is the perfect way to teach remotely, create as many courses as you have topics to teach.
+
+.. image:: media/elearning-overview.png
+   :align: center
+   :alt: Elearning Overveview
```

Comment:
overveveveview :p 
 Beside, I'll let @kbr-odoo give you her insights on the role of the `alt` tag since she explains that way better than I do.